src/olympia/amo/tests/test_celery.py

```diff
@@ -1,67 +1,129 @@
+import time
 import datetime
 
 from datetime import timedelta
+from unittest import mock
 
+from django.conf import settings
 from django.core.signals import request_finished, request_started
 from django.test.testcases import TransactionTestCase
 
-from unittest import mock
+import pytest
 
 from post_request_task.task import _discard_tasks, _stop_queuing_tasks
 
 from olympia.amo.celery import task
-from olympia.amo.tests import TestCase
 from olympia.amo.utils import utc_millesecs_from_epoch
+from olympia.amo.tests import wait_for_tasks
 
 
 fake_task_func = mock.Mock()
 
 
+def test_celery_routes_in_queues():
+    queues_in_queues = set([q.name for q in settings.CELERY_TASK_QUEUES])
+
+    # check the default queue is defined in CELERY_QUEUES
+    assert settings.CELERY_TASK_DEFAULT_QUEUE in queues_in_queues
+
+    # then remove it as it won't be in CELERY_ROUTES
+    queues_in_queues.remove(settings.CELERY_TASK_DEFAULT_QUEUE)
+
+    queues_in_routes = set(
+        [c['queue'] for c in settings.CELERY_TASK_ROUTES.values()])
+    assert queues_in_queues == queues_in_routes
+
+
+@task(ignore_result=False)
+def fake_task():
```

Comment:
Most tasks here are "fake" but this one is probably an `ignore_result_task` (given that the other one is the default task, because it has no task options).